src/MemcacheCache.php

```diff
@@ -14,6 +14,9 @@
 
 use Memcache;
 
+/**
+ * @deprecated
```

Comment:
Why? What to use instead?

Reply:
Because of this: https://github.com/doctrine/cache/commit/9e431040313de8fee56b9c6065e28edf51b0a8b0#diff-d064a32c10f109533c0de3557cee8922

Kdyby\DoctrineCache\MemcacheCache is a descendant of \Doctrine\Common\Cache\MemcacheCache, which has been deprecated a year ago.
I believe you should use MemcachedCache instead, which has an ongoing development. Memcache died like 6 years ago and it's very hard (if possible) to run it with php7+ (https://bugs.php.net/bug.php?id=72887)